src/applications/burials/tests/helpers.unit.spec.jsx

```diff
@@ -12,7 +12,7 @@ import {
 describe('Burials helpers', () => {
   describe('submit', () => {
     beforeEach(() => {
-      window.VetsGov = { pollTimeout: 1 };
+      window.VetsGov.pollTimeout = 1;
```

Comment:
There's one place in the app that uses the scroll params from `VetsGov`, and these pollTimeout lines were previously overwriting the code we put in place to set the scroll delays to 0.